301-availability-set-elastic-storage-ecs/azuredeploy.parameters.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "osImageOffer": {
```

Comment:
remove these and use defaultValues in the template

Reply:
ok